packages/plugin-core/src/survey.ts

```diff
@@ -607,19 +608,13 @@ export class SurveyUtils {
             "https://airtable.com/shry4eLgvVE6WR0Or?prefill_SurveyName=InactiveFeedback";
           VSCodeUtils.openLink(AIRTABLE_URL);
 
-          await StateService.instance().updateGlobalState(
-            GLOBAL_STATE.INACTIVE_USER_SURVEY_SUBMITTED,
-            "submitted"
-          );
+          MetadataService.instance().setInactiveUserMsgStatus("submitted");
```

Comment:
can we also deprecate `StateService`? i'm suspect of all data from it - we should really be moving everything into `MetadataService`

Reply:
I'll mark it as deprecated for now, and add it to [[Impactful Change Notice|dendron://dendron.docs/dev.ref.impactful-change-notice]] (+async)

Reply:
- notice: [[Deprecation of StateService|dendron://dendron.docs/dev.ref.impactful-change-notice#deprecation-of-stateservice]]
- task: [[Deprecate Stateservice and Migrate to Metadataservice|dendron://private/task.2022.03.14.deprecate-stateservice-and-migrate-to-metadataservice]]